pam/internal/adapter/textinputmodel.go

```diff
@@ -10,6 +10,17 @@ type textinputModel struct {
 	textinput.Model
 }
 
+func newTextInputModel(entryType string) textinputModel {
+	inputModel := textinputModel{Model: textinput.New()}
+
+	switch entryType {
+	case "chars_password", "digits_password":
```

Comment:
We handle case "digits_password" here but we never call `newTextInputModel` with that type.

Reply:
Well, not yet but it's supported, so since we're building a generic model here I think it's fine to have it around.